.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,31 +0,0 @@
----
-name: 🐛 Bug report
-about: File a bug report
-title: ''
-labels: community, bug
-assignees: ''
-
----
-
-**Current behaviour**
-<!-- What is be happening. -->
-
-**Expected behaviour**
-<!-- What should be happening. -->
-
-**Steps to reproduce**
-<!--
-  How can we reproduce this issue in order to diagnose it?
-  Code snippets, log messages, screenshots and sample apps are encouraged!
--->
-
-**How does `datadog` help you?**
-<!-- Optionally, tell us why and how you're using datadog, and what your overall experience with it is! -->
```

Comment:
I can add it, it's certainly easy to do, but I'm not sure if an issue report is the appropriate place to collect this information. Is it actionable? None of the other tracers ask for it. It's also not Ruby specific.

Reply:
So... I argue that not everything needs to exist for a purely functional purpose. That's why our offices have great views and cool art on the wall ;)

_/me drags over his soap box, gets on top of it_

An issue report is often something that can sound quite negative/disappointing. Customers are coming to us and saying "hey, this thing is breaking my app! I'm unable to get this to work! This is not doing what I wanted".

Having this section is an **optional** opportunity to mix it up a bit. To pause and say "yeah, I've been a very happy user so far, and I use it every day" kinda thing.

Here's 5 examples of customers filling it:

**How does datadog help you?**

> We use it for tracing, logging, CI management and security (`https://github.com/DataDog/dd-trace-rb/issues/4191`)

> We are using the Ruby profiler as part of our observability stack to profile Ruby applications. (`https://github.com/DataDog/dd-trace-rb/issues/3853`)

> I'm trying to understand a trace with lots of cache spans but can't determine where to improve the application because the span is missing the necessary information. (`https://github.com/DataDog/dd-trace-rb/issues/3808`)

> Some great tracing and profiling :) (`https://github.com/DataDog/dd-trace-rb/issues/3804`)

> My team and I benefit immensely from Datadog observability! Quick debugging, performance optimization and proactive alerting. (`https://github.com/DataDog/dd-trace-rb/issues/3784`)

> To extend our log by helpful information to support customers (`https://github.com/DataDog/dd-trace-rb/issues/3773`)

Could we have tackled these issues without this info? Yeah, most definitely. And yet... it's nice to read these, and sometimes even useful because it hints at the wider use-case, most definitely!

Every opportunity we have to interact with a customer is an opportunity to open a dialog, to understand what's good and bad about our software, to see how we can make them more successful and also of course to sell our product and brand!

We're not computers, and neither are our customers, so in my humble opinion we shouldn't over-optimize to treat this process like a computational one. This is a conversation, between humans, not an API call between computers!

_/me puts away his soapbox_ and fades away

Reply:
This field has been added back to the issue template